luma/core/interface/ftdi.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Richard Hull
+# See LICENSE.rst for details.
+
+
```

Comment:
docstring inc version added info is missing.

Reply:
These aren't meant to be for public consumption, so I might just inline them into the serial.py file

Reply:
the fact they're not meant for public consumption does not mean it's still useful to know when they were added IMO.